int32 num_blocks
---
geometry_msgs/Pose[] poses